overlord/devicestate/devicestate_bootconfig_test.go

```diff
@@ -68,6 +70,14 @@ func (s *deviceMgrBootconfigSuite) SetUpTest(c *C) {
 	})
 	s.state.Set("seeded", true)
 
+	const pcGadgetSnapYaml = `
+name: pc
+type: gadget
+`
+	s.gadgetSnapInfo = snaptest.MockSnapWithFiles(c, pcGadgetSnapYaml, si, [][]string{
+		{"meta/gadget.yaml", gadgetYaml},
+	})
```

Comment:
do we need also a new test where the new gadget comes with  a cmdline.extra?

Reply:
This is done now.